docs/superpowers/specs/2026-05-16-umami-analytics-adapter-design.md

```
@@ -0,0 +1,289 @@
+		"data": { "id": "1_00" }
+	}
+}
+```
+
+Source: <https://docs.umami.is/docs/api/sending-stats>
+
+## Architecture
+
+Three layers under `src/lib/Analytics/`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)